packages/itwinui-css/src/progress-indicator/overlay.scss

```diff
@@ -12,33 +12,27 @@
   top: 0;
   left: 0;
   z-index: 10000;
-  pointer-events: none;
 
   // Blur fallback
   @include iui-blur($hsl: var(--iui-color-background-hsl), $opacity: 3);
 
-  // Focus trap
-  &:focus {
-    outline: 0;
-  }
-
   .iui-progress-indicator-linear {
     width: 50%;
     max-width: 33vw;
   }
+}
 
-  // Fade out on completion
-  @keyframes closeAnimation {
-    from {
-      opacity: var(--iui-opacity-1);
-    }
-
-    to {
-      opacity: 0;
-    }
+// Fade out on completion
+@keyframes closeAnimation {
+  from {
+    opacity: var(--iui-opacity-1);
   }
 
-  .iui-overlay-exiting {
-    animation: closeAnimation var(--iui-duration-1) linear;
+  to {
+    opacity: 0;
   }
 }
+
+@mixin iui-progress-overlay-exiting {
```

Comment:
Just curious, why did you make this class into a plugin? Just better code or something else?

Reply:
Did you read the PR description?